README.md

```diff
@@ -99,6 +99,27 @@ $httpClient = $this->container->get('httplug.client.my_guzzle5');
 $httpClient = $this->container->get('httplug.client.acme');
 ```
 
+#### Plugins
+
+You can configure the clients with plugins. 
+
+```yaml
+// services.yml
+my_cache_plugin:
+  class: Http\Client\Plugin\CachePlugin 
```

Comment:
i think we should use a fictional plugin class name here, as we provide the standard plugins as services.